translations/src/config.js

```diff
@@ -34,20 +34,18 @@ class Config {
     tr: { rtl: false },
     pl: { rtl: false },
     ti: { rtl: false },
-    ku: {
-      rtl: false,
+    'ckb-Arab': {
```

Comment:
Should we maybe go here with just ckb? I'm not 100% whether that tag is good. Never seen -Arab

Reply:
It's the same as -Latn, i.e. ISO1592 (https://en.wikipedia.org/wiki/ISO_15924). As long as we don't have latin ckb it would be fine as well, but not sure.

Reply:
@Taggotty please don't loose your mind and go crazy over this mess of language codes and tags and stuff :pray: :grin:

Reply:
Hmm. I think we need to make a decision here. After reading through this article about the kurdish alphabets and this article about the Sorani language, I feel like Central Kurdish still heavily relies on the Persian alphabet. At least at school or at home they seem to learn this alphabet. The latin alphabet might be an adaption to make Central Kurdish more accessible to foreigners to learn their language, similar to the Romaji in Japan (which is also learned at school there, but no native person would use it). In that case it might be quite strange to add translations for `ckb-Latn`. Therefore, I think this part really needs some more discussion and more expertise than we can attain from reading through wikipedia articles... But for the other version using the persian alphabet, I think we can agree to implement it, using the `ckb` tag, as already discussed.

Some other references about the alphabet:
https://www.mustgo.com/worldlanguages/kurdish/#writing
https://sites.fas.harvard.edu/~iranian/Sorani/sorani_1_grammar.pdf
Kurdish news pages also seem to only use the arab alphabet:
https://www.kurdistan24.net/ckb
https://kurdistantv.net/index.php/ku (here ku seems to mean ckb)

Reply:
And to throw everyone off i also found this reddit post on this: https://www.reddit.com/r/kurdistan/comments/ax883g/sorani_written_in_latin_alphabet/?utm_source=share&utm_medium=web2x&context=3

Reply:
:smile: nice, no we have got all possible variants I guess... There is also a kmr version though: D https://www.kurdistan24.net/kmr
But I think you are right and `ckb` is (nearly) exclusively arabic/persian font, so `kmr´ and `ckb` should be enough. As long as it is clear for translators, cities and ourselves what each tag means I think we are good.